python/ccf/cose.py

```diff
@@ -28,8 +28,13 @@
 Pem = str
 
 GOV_MSG_TYPES_WITH_PROPOSAL_ID = ["ballot", "withdrawal"]
+GOV_MSG_TYPES_WITH_MEMBER_ID = ["recovery_share"]
 
-GOV_MSG_TYPES = ["proposal", "ack", "state_digest"] + GOV_MSG_TYPES_WITH_PROPOSAL_ID
+GOV_MSG_TYPES = (
+    ["proposal", "ack", "state_digest"]
+    + GOV_MSG_TYPES_WITH_PROPOSAL_ID
+    + GOV_MSG_TYPES_WITH_MEMBER_ID
```

Comment:
We don't need this, members are identified by the kid.

Reply:
We still need this added to `GOV_MSG_TYPES` here on the client side, so that we can call `ccf_cose_sign1 --ccf-gov-msg-type recovery_share`, and not get a `argument --ccf-gov-msg-type: invalid choice: 'recovery_share'` error. I've removed the additional plumbing for a `ccf.gov.msg.message_id` header+arg.